kratos/includes/properties.h

```diff
@@ -239,7 +239,7 @@ class Properties : public IndexedObject
     template<class TXVariableType, class TYVariableType>
     TableType const& GetTable(const TXVariableType& XVariable, const TYVariableType& YVariable) const
     {
-                return mTables.at(Key(XVariable.Key(), YVariable.Key()));
+        return mTables[Key(XVariable.Key(), YVariable.Key())];
```

Comment:
Here you are changing the behavior when table does not exist. And `operator[]` does not have a const version so I'm not sure if it is portable....

Reply:
The problem here was that `PointerVectorMap` does not have the `at` method.

Could we manually add a check in case the requested table does not exist?

Reply:
Or maybe you could implement the `at` method in the `PointerVectorMap`?
I looked at it but that is (for now) out of my league

Reply:
So how it was compiling until now!? Nobody was using this method?

Reply:
Now I see, this is due to the change in tables container

Reply:
I have added the at method to the PointerVectorMap but would you please try to use std::unordered_map instead of PointerVectorMap? It should be faster and more robust...